<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>not_islemleri</class>
 <widget class="QWidget" name="not_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_tumogr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tumders">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>50</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tumogrders">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>490</y>
     <width>951</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumogr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumders">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumogrders">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kod">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_vize">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_final">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_no">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kod">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vize">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_final">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ekle">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>440</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
